### PR TITLE
feat : add avatar component

### PR DESCRIPTION
**Pull request by Kavita539**
Added avatar component.
It includes two types of avatar - Profile Avatar and text Avatar. Available in 4 different sizes from xs to large.
Made the iframe responsive.

Layout of Avatar Component (Profile Avatar):
image

 Layout of Avatar Component (Text Avatar):
image

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]
Looks good to me

**Review by KanekarSnehal — APPROVED** [review]
Well Structured👍

**Review by Vivek-Py — APPROVED** [review]
Looks good to me 🚀

File: stylesheets/layout/code-container.css [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-    width: 97%;
-    overflow: hidden;
-    margin: 2rem auto;
-    padding: 1rem;
-    background-color: var(--code-container-color);
-    border-radius: 0.3rem;
+  position: relative;
+  padding-top: 56.25%;
+  /* padding-top: 30px; */
+  /* padding: 12rem 0; */
```
Comment (tewarimohit) [inline stylesheets/layout/code-container.css:7]: remove unnecessary comments.

File: stylesheets/layout/code-container.css [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+  margin: 2rem 0;
+  width: 100%;
+  /* height: 0; */
```
Comment (tewarimohit) [inline stylesheets/layout/code-container.css:10]: remove if not using.

File: stylesheets/layout/code-container.css [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
+  overflow: auto;
+  /* background-color: var(--code-container-color); */
```
Comment (tewarimohit) [inline stylesheets/layout/code-container.css:12]: remove if not using this.

**Review by tewarimohit — COMMENTED** [review]
Please go through the mentioned changes in css file.

**Pull request merged**